doc/nrf-bm/libraries/bluetooth/peer_manager.rst

```diff
@@ -0,0 +1,414 @@
+         LOG_ERR("pm_register() failed, err: 0x%x", err);
+         return -EFAULT;
+      }
+   }
+
+Usage
+*****
+
+Security parameters
+===================
+
+The :c:func:`pm_sec_params_set` function configures how the Peer Manager behaves when securing the link, thus it configures bonding, pairing, and encryption.
+The configuration is given by security parameters (:c:struct:`ble_gap_sec_params_t`).
+These security parameters are also used directly in the SoftDevice security API and contain the parameters that are sent over-the-air during the bonding procedure.
+See the `Bluetooth Core Specification`_ (sections 3.H.3.5.1 and 3.H.3.5.2) for more information.
+
+The :c:func:`pm_sec_params_set` function rejects invalid security parameters.
+See the mentioned Bluetooth specification sections or the verification function in the Peer Manager source code for the constraints on the parameters.
+
+The following list shows the required security parameters for common use cases:
+
+* No pairing/bonding: ``NULL`` (or do not call the :c:func:`pm_sec_params_set` function)
+* Pairing, no bonding:
+
+.. code-block:: c
+
+   sec_param.bond = false;
+   sec_param.mitm = false;
+   sec_param.lesc = 0;
+   sec_param.keypress = 0;
+   sec_param.io_caps = BLE_GAP_IO_CAPS_NONE;
+   sec_param.oob = false;
+   sec_param.min_key_size = 7;
+   sec_param.max_key_size = 16;
+   sec_param.kdist_own.enc = 0;
+   sec_param.kdist_own.id = 0;
+   sec_param.kdist_peer.enc = 0;
+   sec_param.kdist_peer.id = 0;
+
+* Just Works bonding:
+
+.. code-block:: c
+
+   sec_param.bond = true;
+   sec_param.mitm = false;
+   sec_param.lesc = 0;
+   sec_param.keypress = 0;
+   sec_param.io_caps = BLE_GAP_IO_CAPS_NONE;
+   sec_param.oob = false;
+   sec_param.min_key_size = 7;
+   sec_param.max_key_size = 16;
+   sec_param.kdist_own.enc = 1;
+   sec_param.kdist_own.id = 1;
+   sec_param.kdist_peer.enc = 1;
+   sec_param.kdist_peer.id = 1;
+
+* Passkey bonding with keyboard capabilities:
+
+.. code-block:: c
+
+   sec_param.bond = true;
+   sec_param.mitm = true;
+   sec_param.lesc = 0;
+   sec_param.keypress = 0;
+   sec_param.io_caps = BLE_GAP_IO_CAPS_KEYBOARD_ONLY;
+   sec_param.oob = false;
+
+The rest of the parameters are the same as for Just Works bonding.
+
+* OOB bonding:
+
+.. code-block:: c
+
+   sec_param.bond = true;
+   sec_param.mitm = true;
+   sec_param.lesc = 0;
+   sec_param.keypress = 0;
+   sec_param.io_caps = BLE_GAP_IO_CAPS_NONE;
+   sec_param.oob = true;
+
+The rest of the parameters are the same as for Just Works bonding.
+
+* Disallow IRKs:
+
+.. code-block:: c
+
+   sec_param.bond = true;
+   /* Arbitrary parameters removed. */
+   sec_param.kdist_own.enc = 1;
+   sec_param.kdist_own.id = 0;
+   sec_param.kdist_peer.enc = 1;
+   sec_param.kdist_peer.id = 0;
+
+Event handling
+==============
+
+How to handle Peer Manager events depends on the application.
+The Peer Manager provides different kinds of events; some must be handled, while others can be disregarded.
```

Comment:
```suggestion
The Peer Manager provides different kinds of events.
Some events must be handled, while others can be disregarded.
```

Reply:
We should state which events that must be handled here (or which event handlers that must be registered).

Reply:
Sentences will be removed at next push. We can improve and be more explicit later.